src/robots/upper_body.cpp

```diff
@@ -254,6 +287,44 @@ class UpperBody: public cardsflow::kindyn::Robot{
 
     }
 
+    void publishTarget(string body_part, bool zeroes){
```

Comment:
add an inline comment what this does, it's unclear by the function signature what is actually happening